examples/rng/device/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ foreach(rng_device_source ${RNG_DEVICE_SOURCES})
       ONEMKL::SYCL::SYCL
   )
 
+  if(NOT ${ONEMKL_SYCL_IMPLEMENTATION} STREQUAL "hipsycl")
```

Comment:
I think it would be more robust to use `is_dpcpp` in these `if` statements. That variable is set if `-fsycl` is supported which seems to be what you want here.

Reply:
Hi Romain,
Unfortunately, `is_dpcpp` flag value is empty on windows. `check_cxx_compiler_flag("-fsycl" is_dpcpp)` fails and sets nothing. We're investigating why it's so. It may be a compiler bug. That's why I reverted these changes otherwise there will be failures on windows again

Reply:
I see, that's not a bug `find_package(Compiler)` is never called on Windows unfortunately: https://github.com/oneapi-src/oneMKL/blob/develop/CMakeLists.txt#L270